hydroflow/tests/surface_codegen.rs

```diff
@@ -2,8 +2,7 @@ use std::collections::{HashMap, HashSet};
 
```

Comment:
why these changes? is `recv_into` deprecated or are you trying to demonstrate something different, or...?

Reply:
Originally there were two methods, `recv_into` and `collect_ready`, both of which collecting ready items from an async stream and put them into a collection. However `recv_into` only worked on channel recv streams. Meanwhile `collect_ready` was async even though it didn't really need to be. So they are now both combined into one `collect_ready` which is synchronous and handles all stream types